docs/azure/sdk/aspnetcore-guidance.md

```diff
@@ -0,0 +1,307 @@
+
+    // Set up any default settings
+    clientBuilder.ConfigureDefaults(
+        builder.Configuration.GetSection("AzureDefaults"));
+});
+```
+
+In the preceding code, the `clientBuilder.UseCredential()` method accepts an instance of `DefaultAzureCredential` that will be reused across your registered services. `DefaultAzureCredential` discovers available credentials in the current environment and use them to connect to Azure services. The full order and locations in which `DefaultAzureCredential` looks for credentials can be found in the [`Azure Identity library overview`](/dotnet/api/overview/azure/Identity-readme#defaultazurecredential).
+
+For example, when you run the app locally, `DefaultAzureCredential` discovers and uses credentials from the following developer tools:
+
+- Environment variables
+- Visual Studio
+- Azure CLI
+- Azure PowerShell
+- Azure Developer CLI
+
+`DefaultAzureCredential` also discovers credentials after you deploy your app from the following:
+
+- Environment variables
+- Workload identity
+- Managed identity
+
+## Configure logging
+
+The Azure SDK for .NET client libraries include the ability to log client library operations. This logging allows you to monitor requests and responses between services clients and Azure services. When you register the Azure SDK library's client via a call to the <xref:Microsoft.Extensions.Azure.AzureClientServiceCollectionExtensions.AddAzureClients%2A> extension method, some logging configurations are handled for you.
+
+```csharp
+    ```csharp
+builder.Services.AddAzureClients(clientBuilder =>
+{
+    clientBuilder.AddSecretClient(
+        builder.Configuration.GetSection("KeyVault"));
+
+    clientBuilder.AddBlobServiceClient(
+        builder.Configuration.GetSection("Storage"));
+
+    clientBuilder.AddServiceBusClientWithNamespace(
+        builder.Configuration["ServiceBus:Namespace"]);
+
+    clientBuilder.UseCredential(new DefaultAzureCredential());
+
+    // Set up any default settings
+    clientBuilder.ConfigureDefaults(
+        builder.Configuration.GetSection("AzureDefaults"));
+});
+```
+
+In the preceding sample, the `AddAzureClients` method:
+
+- Registers the following objects with the dependency injection (DI) container:
+  - Log forwarder service
+  - Azure Service Bus client
+- Sets the default token credential to be used for all registered clients.
+
+You can change default log levels and other settings using the same JSON configurations outlined in the [configure authentication](#configure-authentication) section. For example, toggle a the `ServiceBusClient` log level to `Debug` by setting the `Logging:LogLevel:Azure.Messaging.ServiceBus` key as follows:
+
+```json
+{
+"Logging": {
+        "LogLevel": {
+            "Default": "Information",
+            "Microsoft.AspNetCore": "Warning",
+            "Azure.Messaging.ServiceBus": "Debug"
+        }
+    }
+}
+```
+
+## Unit testing considerations
```

Comment:
As discussed via Teams, remove this section. There's nothing specific to ASP.NET Core as it relates to unit testing.